changes/8684.feature

```diff
@@ -0,0 +1 @@
+`datastore_create` now accepts `include_records` to return inserted records in the return dict. `datastore_create`, `datastore_upsert`, and `datastore_delete` will now include the SQL rows from `RETURNING` in the return dict's `records` instead of the user passed list.
```

Comment:
indicate that the records will now *not* be returned unless include_records=True is passed and that the values that *used* to be returned were not necessarily the actual values stored, but are now.

Reply:
changelog updated to be better. hopefully??